pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -64,3 +64,16 @@ func BindingSpecChecksum(spec servicecatalog.BindingSpec) string {
 	sum := sha256.Sum256([]byte(specString))
 	return fmt.Sprintf("%x", sum)
 }
+
+// BrokerSpecChecksum calculates a sha256 hash for the given BrokerSpec based on
+// the following fields:
+// - URL
+// - AuthInfo (may be nil, but special handling is unnecessary with %v)
```

Comment:
What does `%v` do with special handling?  Can we put a toplevel comment at the top?